### ISSUE TITLE
Allow unlimited alerts for those with .edu emails

### ISSUE THREAD

**Issue by mlissner** [freelawproject/foresight#29]
A couple reasons to do this:

1. It should be fairly easy.
2. It'll give librarian-types another reason to mention us in their guides.
3. It'll get people using our systems early.
4. Everybody else does this, I think, so we should too.

I think there are two technical ways to go about it:

1. Give them free membership benefits at a particular level if they have .edu email addresses. We're looking into how feasible this is in Neon.

5. Just provide anybody that logs in CL with member benefits, leaving Neon out of it. 

I prefer number one, if we can do it:

 - It involves no extra code in CL
 - It provides one flow for our users: Become a member -> Get benefits
 - It ensures that .edu people always get the full benefits package
 - It reinforces that we're a non-profit, which is good stewardship of the next generation of donors. :)

We'll follow up with whether option one is feasible!

**Comment by anseljh** [comment]
Just noting that a lot of alumni use an `.edu` address. I have an `@alum.berkeley.edu` one, and I haven't been a student there in more than 20 years.

**Comment by mlissner** [comment]
Yeah, that's fine. If you're organized enough to keep your alum email working, fine, you win. :)
